### DISCUSSION TITLE
Share your Drive Stats speed histogram!

### DISCUSSION THREAD

**Discussion by eliluong** [teslamate-org/teslamate#4330] [General]
I like the speed histogram widget on the dashboards! Curious what everyone's looks like and if you have a story behind it. I am from Southern California, so lots of traffic all the time. Sometimes crawling along that it takes 10-15min to get one mile on the freeway. Sad that most of my time is spent driving < 10 mph.

![image](https://github.com/user-attachments/assets/70bde1a0-aeeb-4491-84bd-55ae7e5b6ba4)

**Comment by JakobLichterfeld** [comment]
I wonder why I never got to 264 km/h:
![image](https://github.com/user-attachments/assets/9f1a9e8e-e4a5-410c-a550-8a68de8cdc4e)

In recent times mostly driving through the city with 40 km/h speed limit:
![image](https://github.com/user-attachments/assets/de663892-0c23-440b-9919-bbbfe72b27ec)
